src/database/common.c

```diff
@@ -221,6 +221,17 @@ void db_init(void)
 		return;
 	}
 
+	// Initialize database lock mutex
+	if (pthread_mutex_init(&dblock, NULL) != 0)
+	{
+		logg("FATAL: FTL_db mutex init failed\n");
```

Comment:
Is there some error output we can use to know why the mutex failed to initialize?